File: modulo1/segmento3/apunte6/Ejercicio1.java
import java.util.Scanner;

public class Ejercicio1 {
	public static void main(String args[]){
		Scanner keyboard = new Scanner(System.in);
		Integer number = 0;

		while(number <= 100) {
			System.out.print("Ingrese un numero : ");
			number = Integer.parseInt(keyboard.nextLine());
		}
	}
}

File: modulo1/segmento3/apunte7/Ejercicio1.web/ejercicio1.js
function divisores(number){

	var cantDivisores = 0;

	for (i=1; i <= number; i++){
		if(number%i == 0){
			cantDivisores += 1;
		}
	}
	return cantDivisores;
	}


	//MAIN 
	function ejercicio1(){
	
	var number = parseInt(prompt('Ingrese un numero : '));
	
	cantDivisores = divisores(number);

	alert('El numero ' + number + ' tiene : '+ cantDivisores + ' divisores');
}